man/namedlist.Rd

```diff
@@ -0,0 +1,12 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
It seems wrong that this is documented - it's not supposed to be exposed as part of public API, it's just for internal convenience.